src/github/index.ts

```diff
@@ -49,15 +48,8 @@ export interface OctokitError extends Error {
 }
 
 export interface GitHubAPI extends Octokit {
+  request: (Route: string, RequestOptions: RequestOptions) => Promise<Octokit.AnyResponse>
   paginate: (res: Promise<Octokit.AnyResponse>, callback: (response: Promise<Octokit.AnyResponse>, done?: () => void) => void) => Promise<any[]>
-  // The following are added because Octokit does not expose the hook.error, hook.before, and hook.after methods
-  hook: {
-    error: (when: 'request', callback: (error: OctokitError, options: RequestOptions) => void) => void
-    before: (when: 'request', callback: (result: Result, options: RequestOptions) => void) => void
-    after: (when: 'request', callback: (result: Result, options: RequestOptions) => void) => void
-  }
-
-  request: (RequestOptions: RequestOptions) => Promise<Octokit.AnyResponse>
   query: (query: string, variables?: Variables, headers?: Headers) => Promise<any>
 }
```

Comment:
I think we should be able to move most of these into octokit/rest.js now? 

Only query and paginate will need overrides so to not introduce breaking changes.

Reply:
Yep good callout. Going to try and focus on this tomorrow.